src/main/java/frc/robot/subsystems/climber/ClimberConstants.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.climber;
+
+public class ClimberConstants {
+
+  private static final String CONSTRUCTOR_EXCEPTION = "constant class";
+
+  private ClimberConstants() {
+    throw new IllegalStateException(CONSTRUCTOR_EXCEPTION);
+  }
+
+  public static final boolean DEBUGGING = false;
+  public static final boolean TESTING = false;
+  public static final String CLIMBER = "Climber";
+
+  // FIXME: update can ids
+  public static final int LEFT_MOTOR_CAN_ID = 20;
+  public static final int RIGHT_MOTOR_CAN_ID = 20;
+
+  // FIXME: update gear ratio
+  public static final double GEAR_RATIO = 25.0;
+  public static final boolean MOTOR_INVERTED = false;
+
+  public static final double LEFT_POSITION_PID_P = 0.0;
+  public static final double LEFT_POSITION_PID_I = 0;
+  public static final double LEFT_POSITION_PID_D = 0;
+  public static final double LEFT_POSITION_PID_KS = 1.0;
```

Comment:
We will also need KV, KA, Expo kA, Expo kV constants for Motion Magic Expo position control.